tests/example_app/schema.py

```diff
@@ -39,5 +39,13 @@ class Mutation:
     delete_cookie = jwt_mutations.DeleteJSONWebTokenCookie.delete_cookie
 
 
-schema = Schema(query=Query, mutation=Mutation, extensions=[AsyncJSONWebTokenMiddleware])
+@strawberry.type
```

Comment:
Is that required or just for readability?

Reply:
RefreshAsync will be required, the refresh token db access is done inside the resolver so it must be wrapped in sync_to_async. Obtain handles this in decorators where it is already async safe.

Reply:
Other are just for consistency